.github/workflows/starknet-rpc-tests.yml

```diff
@@ -31,11 +31,19 @@ jobs:
       - name: Setup dev chain
         run: |
           ./target/release/madara setup --chain=dev --from-local=configs
-      - name: Run rpc test
+      - name: Run rpc test without cache
         run: |-
           ./target/release/madara --dev --sealing=manual &
           MADARA_RUN_PID=$!
           while ! echo exit | nc localhost 9944; do sleep 1; done
           cd starknet-rpc-test
           cargo test
           kill $MADARA_RUN_PID
+      - name: Run rpc test with cache
```

Comment:
Is it really necessary?

Reply:
Our CI is already quite long

Reply:
Ya I agree its long as well. @EvolveArt do you have any views?

Reply:
well it's definitely possible that the cache flag breaks stuff. If we have to choose one path I'd rather go for with cache only then.